src/utils/api.js

```diff
@@ -4,8 +4,12 @@ const newsApi = axios.create({
     baseURL: "https://nc-news-dw.onrender.com/api"
 })
 
-export const fetchArticles = () => {
-    return newsApi.get(`/articles`)
+export const fetchArticles = (topic_name) => {
+    const params = {}
```

Comment:
This is fine, but the way Axios params works it's also safe to have the property in the params object and when the value is undefined, it will just get ignored and you will receive all the articles.